### ISSUE TITLE
friendly name for weather entity

### ISSUE THREAD

**Issue by Mariusthvdb** [briis/weatherbit#44]
Hi!

please consider naming the weather entity simply Weatherbit? as it stands we need to do:
```
  - type: weather-forecast
    entity: weather.weatherbit_day_based_forecast
    name: Weatherbit
```
to not show the extended forecast name in the frontend. Not a big detail, but could be easier ;-)

btw, working nicely, so congrats on the new integration

**Comment by briis** [comment]
Makes sense. I think it was named like this, because I was working on the WeatherFlow integration at the same time, and here I can pull both a Day and Hourly forecast. I will rename, which then will be a breaking change.

**Comment by Mariusthvdb** [comment]
just to be sure, I also meant the entity name, changing to `weather.weatherbit` with name: Weatherbit

**Comment by briis** [comment]
I know 😁 Will be done in a minute or so.

**Comment by briis** [comment]
Released 1.0.1 where this is included.
